### ISSUE TITLE
Background image not displaying when selecting games list after "firm update" from .6 to .7

### ISSUE THREAD

**Issue by Skyway1985** [tpunix/SAROO#220]
Small issue, but an issue non the less. After updating from .6 to .7 my background image only displays in the Main Menu. When in the game list, the background image doesn't load. Did somehting change? are you now able to have a seperate image for main menu and game list? if so, what should you name the GIF? other than that, i've had no issues as of yet!

**Comment by tonysantosl** [comment]
yeah, I thought this was an issue only with animated gifs but it seems to be happening with any image set as bg.

I wonder if @tpunix could also make the menu font smaller in the future so the box art don't cover the text.

**Comment by Skyway1985** [comment]
Well at least you can press Dpad left to make the cover fall into the background, making it easy to read the whole title. When you back out to the main menu, the GIF starts again...... is it the same for you?
